Qast/Managers/FirebaseManager.swift

```diff
@@ -1,43 +1,50 @@
-import Foundation
-import Firebase
+//
+//  FirebaseManager.swift
+//  Qast
+//
+//  Created by Austin Welch on 7/5/19.
+//  Copyright © 2019 Qast. All rights reserved.
+//
+
 import FirebaseFirestore
 import CoreLocation.CLLocation
 
-let DB_BASE = Firestore.firestore()
-
-typealias SoundZoneCompletionBlock = ([SoundZone], Error?) -> Void
+struct FirebaseManager: SoundZoneAPI {
 
-class FirebaseManager {
```

Comment:
I'm fine deleting these. I'm curious what you disliked about having a group of constants for Firestore refs though? Just not necessary at the moment?

Reply:
We don't want globally accessible variables if we don't need them. And I think that Firestore.firestore() handles all config for us

Reply:
Firestore.firestore() does do all of the config for fetching your DB_BASE.

These properties are private, and the getters are there so we can refer to collection paths easier without having many `datastore.collection("string_that_may_change")` throughout this manager.